MAINTAINERS.md

```diff
@@ -0,0 +1,24 @@
+# ODPi Maintainers
+## What is an ODPi Maintainer?
+
+ODPi Maintainers are the key leaders in the development of the ODPi specifications. They are key members of the ODPi community who have shown both leadership and collaboration skills in thier work within the community.
+
+### Roles and Responisibilities
+
+* Review spec and test harness contributions that for the spec(s) you are a maintainer for.
+* Control the direction of the spec(s) you are a maintainer for.
+* Act as the point-of-contact for questions from users and other committers for the spec(s) you are a maintainer for.
+* Adhere to and be an example of the [ODPi Code of Conduct](https://github.com/odpi/specs/wiki/ODPi-Code-of-Conduct)
+* Elect ODPi development community members to the TSC and RT
+
+## How to become an ODPi Maintainer?
+
+ODPi Maintainers are elected by the other ODPi Maintainers. There is no membership requirement with ODPi to become an ODPi Maintainer.
```

Comment:
Are maintainers elected at a set time (like TSC and RT) or is it done anytime an existing maintainer proposes electing a new maintainer?  I would assume the latter.